src/core/data-cite.js

```diff
@@ -37,7 +37,12 @@ function requestLookup(conf) {
       href = entry.href;
     }
     if (path) {
-      href = new URL(path, href).href;
+      // We resolve absolute paths relative to the spec
```

Comment:
maybe just adding a link to the issue (https://github.com/w3c/respec/issues/1856) would be enough?